docs/UserGuide.md

```diff
@@ -32,80 +32,173 @@
+* `add follow up with Jack on sales report sd/6-10-2016`
+* `add project Highlight sd/1-10-2016 ed/14-11-2016`
 
-#### Listing all persons : `list`
-Shows a list of all persons in the address book.<br>
-Format: `list`
+#### Set a deadline to a task : `setdeadline`
+Sets the amount of time left or an end date to the specified task from the task manager.<br>
+Format: `setdeadline TASK_INDEX tl/TIMELEFT` or `setdeadline TASK_INDEX ed/END_DATE`
 
-#### Finding all persons containing any keyword in their name: `find`
-Finds persons whose names contain any of the given keywords.<br>
-Format: `find KEYWORD [MORE_KEYWORDS]`
+> Sets a deadline to the task at the specified `TASK_INDEX`.<br>
+  The index refers to the index number shown in the most recent listing.<br>
+  The index **must be a positive integer** 1, 2, 3, ...
+  
+Examples: 
+* `list`<br>
+  `setdeadline 2 tl/24`<br>
+  Set a deadline with 24 hours left to the 2nd task in the task manager.
+* `find report`<br> 
+  `setdeadline 1 ed/23-10-2016`<br>
+  Set a deadline on 23rd October 2016 to the 1st task in the results of the `find` command.
+
+#### Set an event to a task : `setevent`
+Sets an event to the specified task from the task manager.<br>
+Format: `setevent TASK_INDEX EVENT_NAME [sd/START_DATE] [st/START_TIME] [ed/END_DATE] [et/END_TIME]`
 
-> * The search is case sensitive. e.g `hans` will not match `Hans`
-> * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-> * Only the name is searched.
-> * Only full words will be matched e.g. `Han` will not match `Hans`
-> * Persons matching at least one keyword will be returned (i.e. `OR` search).
-    e.g. `Hans` will match `Hans Bo`
+> Sets an event to the task at the specified `TASK_INDEX`.<br>
+  The index refers to the index number shown in the most recent listing.<br>
+  The index **must be a positive integer** 1, 2, 3, ...
 
 Examples: 
-* `find John`<br>
-  Returns `John Doe` but not `john`
-* `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
+* `list`<br>
+  `setevent 2 complete milestone 1 sd/1-10-2016 st/1400 ed/1-10-2016 et/1600`<br>
+  Set an event on 1st October 2016 starting from 1400 - 1600 to the 2nd task in the task manager.
+* `find Highlight`<br> 
+  `setevent 1 Highlight roadshow sd/23-10-2016 ed/23-10-2016`<br>
+  Set a full day event on 23rd October 2016 to the 1st task in the results of the `find` command.
+  
+#### Set task as completed : `setcomplete`
+Sets an event to the specified task from the task manager.<br>
+Format: `setcomplete TASK_INDEX`
+
+> Sets the task at the specified `TASK_INDEX` as complete.<br>
+  The index refers to the index number shown in the most recent listing.<br>
+  The index **must be a positive integer** 1, 2, 3, ...
+  
+Examples: 
+* `list`<br>
+  `setcomplete 2`<br>
+  Set the 2nd task as complete in the task manager.
+
+#### Listing all tasks due today : `list`
```

Comment:
what if i have to list all tasks in the task manager? which command do i use for that?